metasv/run_age.py

```diff
@@ -172,14 +178,14 @@ def run_age_single(intervals_bed=None, region_list=[], contig_dict={}, reference
                                                              pad=pad, dist_to_expected_bp=dist_to_expected_bp,
                                                              min_del_subalign_len=min_del_subalign_len,
                                                              min_inv_subalign_len=min_inv_subalign_len,
-                                                             age_window=age_window)
+                                                             age_window=age_window, sc_locations=sc_locations)
                 bedtools_fields = matching_interval.fields
                 if len(breakpoints) == 1 and sv_type == "INS":
-                    bedtools_fields += map(str, [breakpoints[0][0], breakpoints[0][0] + 1, breakpoints[0][1]])
+                    bedtools_fields += map(str, [breakpoints[0][0], breakpoints[0][0] + 1, breakpoints[0][1], breakpoints[0][2]])
                 elif len(breakpoints) == 2 and (sv_type in ["DEL","INV","DUP"]):
-                    bedtools_fields += map(str, breakpoints + [breakpoints[1] - breakpoints[0]])
+                    bedtools_fields += map(str, breakpoints + [breakpoints[1] - breakpoints[0]] + ["."])
```

Comment:
Do tuples or list have a subtraction operator? I tried this out in an interactive python session and failed.